lib/improver/utilities/temporal.py

```diff
@@ -31,22 +31,17 @@
 """Provide support utilities for making temporal calculations."""
 
 import re
-
-from datetime import datetime
-from datetime import time as dt_time
-from datetime import timedelta
-from datetime import timezone
 import warnings
-
-import numpy as np
+from datetime import datetime, timedelta, timezone
+from datetime import time as dt_time
```

Comment:
Could just import the module here. 
These could all be used as variable names, seems like unnecessary pollution of the namespace!